server/queue/fifo.go

```diff
@@ -27,6 +27,9 @@ import (
 	"go.woodpecker-ci.org/woodpecker/v2/shared/constant"
 )
 
+// InternalWorkerID is pseudo agent ID for internal routines using the queue.
+const InternalWorkerID = -2
```

Comment:
-2 because -1 is used in the agentID context already ... so just any other negative integer is sufficient